todoapp.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let sub = document.querySelector("#submit");
+  let list = document.getElementById("list");
+
+  let input = document.getElementById("things");
+
+  sub.addEventListener("click", event => {
+    let create_li = document.createElement("li");
+    if (input.value === "") {
+      // let create_par = document.createElement("p");
+      error.innerHTML = "<p>Input somethin'</p>";
+    } else {
+      create_li.innerText = input.value;
+      event.target.parentNode.appendChild(create_li, event.target);
```

Comment:
Because you have the event listener on the submit button, the parent node would the `form` element. Rather than appending the new `li` element to the `form`, we want to append it to our `list` variable, or the `ul` element.